extendeddatacrossword_test.go

```diff
@@ -423,6 +424,186 @@ func TestCorruptedEdsReturnsErrByzantineData_UnorderedShares(t *testing.T) {
 	}
 }
 
+func TestFuzzRandByzantine(t *testing.T) {
+	// This test is slow and should be skipped during normal testing
+	t.Skip()
+	for i := 0; i < 10000; i++ {
+		TestErrRandByzantine(t)
+	}
+}
+
+func TestErrRandByzantine(t *testing.T) {
+	codec := NewLeoRSCodec()
+	original, corrupted, idx := randCorruptedEDS(t, codec, 8)
+	require.False(t, original.Equals(corrupted), "corrupted eds is equal to original eds")
```

Comment:
[question] why `.False` instead of:

```go
	require.NotEqual(t, original, corrupted)
```

Reply:
NotEqual uses `==` slice compare under the hood, while `Equals` method does more precise comparison.